package.json

```diff
@@ -43,11 +43,11 @@
   "bundlesize": [
     {
       "path": "./dist/*.js",
-      "maxSize": "170 kB"
+      "maxSize": "270 kB"
     },
     {
-      "path": "./dist/assets/js/*.js",
-      "maxSize": "170 kB"
+      "path": "./dist/assets/js/googlesitekit-*.js",
+      "maxSize": "270 kB"
     }
```

Comment:
These were more important in the past I believe when the file size of the distributable was a limiting factor for some. Obviously we don't want to ship 50MB of js, but are bundle size tests still necessary?

Reply:
I think it's a good practice to keep these around. We can always adjust as necessary, but it's good to have these even as a flag to us when things might get out of hand.

Reply:
Agreed. We're cool with bumping them now, but it's a nice idiot check in case we accidentally include an extra MB unintentionally. 😄